lib/transport/datagouvfr/client/datasets.ex

```diff
@@ -3,7 +3,50 @@ defmodule Transport.Datagouvfr.Client.Datasets do
   A client to manipulate https://www.data.gouv.fr/api/1/datasets endpoints
   """
 
-  import Transport.Datagouvfr.Client, only: [get_request: 2, put_request: 3]
+  import TransportWeb.Gettext
+  import Transport.Datagouvfr.Client, only: [get_request: 2, put_request: 3,
+                                             post_request: 3, post_request: 4]
+
+  use Vex.Struct
```

Comment:
I'm wondering if all validation logic should rather go into the already defined ```%Dataset{}```.

Reply:
nop